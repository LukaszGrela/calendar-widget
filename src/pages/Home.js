import React from 'react';
import CalendarWidget from '../components/CalendarWidget';

class Home extends React.Component {
    render = () => {
        
        return (
            <section>
                <article>
                    <h1>Calendar Widget</h1>
                    <p>The Code Exercise for JS/React Developer</p>
                </article>
                <article>
                    <CalendarWidget />
                </article>
            </section>
        );
    }
};
export default Home;