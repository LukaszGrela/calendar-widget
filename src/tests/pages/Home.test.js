import React from 'react';
import {shallow} from 'enzyme';
import Home from '../../pages/Home';

test('should render correctly', () => {
    const wrapper = shallow(<Home />);

    expect(wrapper).toMatchSnapshot();
});