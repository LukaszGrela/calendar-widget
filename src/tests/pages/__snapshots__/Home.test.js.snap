// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<section
  className="home"
>
  <article>
    <p>
      The Code Exercise for JS/React Developer
    </p>
  </article>
  <nav>
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      to="/router-calendar/1970/1/1"
    >
      Routed Calendar
    </NavLink>
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      to="/linked-calendars"
    >
      Linked Calendars
    </NavLink>
  </nav>
</section>
`;
